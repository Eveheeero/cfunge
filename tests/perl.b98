"LREP"4(v
v       <
                > a"detcelfeR" >:#,_@
> 0 '" "ih" '" #^ E >:#,_ v
v                         <

> 0 "5+1" I. a, @


This program should print hi6
