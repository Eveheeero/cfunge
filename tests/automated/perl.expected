Hello world!
hi6 
1
5934 
